testsuite/forge-cli/src/main.rs

```diff
@@ -1050,10 +1050,10 @@ fn realistic_env_workload_sweep_test() -> ForgeConfig {
         ]),
         // Investigate/improve to make latency more predictable on different workloads
         criteria: [
-            (5500, 100, 0.3, 0.3, 0.8, 0.65),
-            (4500, 100, 0.3, 0.4, 1.0, 2.0),
-            (2000, 300, 0.3, 0.3, 0.8, 2.0),
```

Comment:
Not sure, but I think it is good to be more conservative and just relax the thresholds in line 1055. (since line 1055 corresponds to the failing test ModifyGlobalResource )